lesson_27_pt2/tests/test_1_po.py

```diff
@@ -1,18 +1,55 @@
+from dotenv import load_dotenv
+import os
 import pytest
+load_dotenv()
+login = os.getenv("LOGIN")
+password = os.getenv("PASSWORD")
+wrong_password = os.getenv("WRONG_PASSWORD")
 
+@pytest.mark.parametrize("login, password", [
+    (os.getenv("LOGIN"), os.getenv("PASSWORD")),
+    (os.getenv("LOGIN"), os.getenv("WRONG_PASSWORD")),
+    ("wrong_mail@com.wrong", "password123")
+])
+def test_sign_in(home_page, cabinet_page, login, password):
+    home_page.login(login, password)
+    if password == os.getenv("PASSWORD"):
+        assert cabinet_page.get_user_profile_element().is_visible(), "User not logged in"
+    else:
+        assert home_page.get_toast_error_element().is_visible(), "Error toast not visible"
 
-def test_homepage_menu(home_page):
+def test_open_empty_cart(home_page,cabinet_page):
+    home_page.login(os.getenv("LOGIN"), os.getenv("PASSWORD"))
```

Comment:
```suggestion
    home_page.login(os.getenv("LOGIN"), os.getenv("PASSWORD"))
```
і знову, або юзайте глобальні зміни або не обявляйте їх